docs/Local_Development_Tutorial.md

```diff
@@ -0,0 +1,168 @@
+# Local Development Tutorial: Getting Started with MarkLogic in Kubernetes
+
+# Table of contents
+* [Introduction](#Introduction)
+* [Prerequisites](##Prerequisites)
+* [Procedure](#Procedure)
+* [Setting Up Minikube](##Setting-Up-Minikube)
+* [Installing MarkLogic to Minikube](##Installing-a-Single-MarkLogic-Host-to-Minikube)
+* [Installing Multiple MarkLogic Hosts to Minikube](##Installing-Multiple-MarkLogic-Hosts-to-Minikube)
+* [Verifying the Installation](##Verifying-the-Installation)
+* [Debugging](#Debugging)
+* [Cleanup](#Cleanup)
+
+# Introduction
+This tutorial describes how to set up local Kubernetes development environment with Minikube and MarkLogic Server. It covers these tasks:
+- Set up the prerequisites necessary for local development using MarkLogic Server in Kubernetes 
+- How to run Minikube and load MarkLogic Server into the Kubernetes cluster 
+- Access the MarkLogic Server cluster
+- How to debug the Kubernetes environment 
+- How to clean up your environment
+
+
+## Prerequisites
+The following steps assume you are running this tutorial from a desktop environment. Mobile environments will likely experience problems and may not work.
+- [Docker](https://docs.docker.com/engine/install/): Subscribe to Docker Hub and pulldown the latest image from: https://hub.docker.com/_/marklogic
+  ```sh
+  # Something similar to this, with the latest version tag, which can be found on the dockerhub link above
+  docker pull store/marklogicdb/marklogic-server:10.0-9-centos-1.0.0-ea4 
+  ```
+- [KubeCTL](https://kubernetes.io/docs/tasks/tools/):  Download and install this tool to assist with debugging in a Kubernetes environment. 
+- [Helm](https://helm.sh/docs/intro/install/):  Clone or download the chart repository: https://github.com/marklogic/marklogic-kubernetes
+- [Minikube](https://k8s-docs.netlify.app/en/docs/tasks/tools/install-minikube/): Minikube Kubernetes environment, which will host the MarkLogic Server application.
```

Comment:
"Download the Minikube Kubernetes environment, which will host the MarkLogic Server applications."